{"name": "Invalid value for param - coords in the Request Body and getting Incorrect output -- @1.3 ", "status": "failed", "statusDetails": {"message": "AssertionError: failed with exception: Got Blank Response as the Output\n", "trace": "  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\pltscisdetstepsimpl.py\", line 78, in verify_response_code_error\n    raise AssertionError(\"failed with exception: \" + str(e))\n"}, "steps": [{"name": "Given I Set POST request api endpoint", "status": "passed", "start": 1628184585126, "stop": 1628184585127}, {"name": "When I Set HEADER param request content type as \"application/json.\"", "status": "passed", "start": 1628184585127, "stop": 1628184585128}, {"name": "When Set invalid request Body { \"roomSize\" : [5, 5], \"coords\" : [1], \"patches\" : [ [1, 0], [2, 2]], \"instructions\" : \"NNESEESWNWW\" }", "status": "passed", "start": 1628184585128, "stop": 1628184585130}, {"name": "And Send a POST HTTP request", "status": "passed", "start": 1628184585130, "stop": 1628184585143}, {"name": "Then I receive error 500, \"Internal Server Error\"", "status": "failed", "statusDetails": {"message": "AssertionError: failed with exception: Got Blank Response as the Output\n", "trace": "  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\pltscisdetstepsimpl.py\", line 78, in verify_response_code_error\n    raise AssertionError(\"failed with exception: \" + str(e))\n"}, "start": 1628184585144, "stop": 1628184585144}], "parameters": [{"name": "request_body", "value": "{ \"roomSize\" : [5, 5], \"coords\" : [1], \"patches\" : [ [1, 0], [2, 2]], \"instructions\" : \"NNESEESWNWW\" }"}, {"name": "response_code", "value": "500"}, {"name": "response_error", "value": "Internal Server Error"}], "start": 1628184585122, "stop": 1628184585146, "uuid": "b3442b8c-d4e5-42e9-bff2-59a0da4fc135", "historyId": "9e484e8729912a0e580a7e7b29ee7328", "fullName": "Test Imaginary Robot for Valid and Invalid Input values for coords params: Invalid value for param - coords in the Request Body and getting Incorrect output", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Test Imaginary Robot for Valid and Invalid Input values for coords params"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}